plugins/linuxplugin/interface_config.go

```diff
@@ -119,15 +122,20 @@ func (plugin *LinuxInterfaceConfigurator) Init(ifIndexes ifaceidx.LinuxIfIndexRW
 
 	plugin.ifWatcherNotifCh = make(chan netlink.LinkUpdate, 10)
 	plugin.ifWatcherDoneCh = make(chan struct{})
-	return nil
+
+	_, plugin.vethCfgNamespace, err = linuxcalls.CreateNamedNetNs("veth-cfg-ns")
```

Comment:
I'm thinking that randomly generated namespace might be better option to avoid collisions. Or at least remove the namespace first if it already exists -- the agent may crash and leave it there with some veths inside (although very unlikely).